actions/eval.go

```diff
@@ -32,6 +33,19 @@ func Eval(c *cli.Context) error {
 	var frm rdef.Formula
 	rhitch.DecodeYaml(f, &frm)
 
+	envVars := c.StringSlice("environment")
+	for _, item := range envVars {
+		splits := strings.SplitN(item, "=", 2)
+		if len(splits) != 2 {
+			fmt.Sprintf("Invalid environment variable '%s' must be of format 'NAME=VALUE'", item)
+			panic("invalid environment variable ")
```

Comment:
Yeah is it just me or is everyone in the US up hella late in preparation for.. easter...?  <.<